<img src="../logo_no_bg.png" alt="Labsinal Logo" width="350"/>

# Labsinal

> üá¨üáß English Version

The [Labsinal ‚Äì Laboratory of Cell Signaling and Plasticity](https://www.ufrgs.br/labsinal/) at the [Federal University of Rio Grande do Sul (UFRGS)](https://www.ufrgs.br/site/) is a multidisciplinary research group composed of biotechnologists, biomedical scientists, biologists, and marine biologists, led by [Prof. Dr. Guido Lenz](http://lattes.cnpq.br/4178667286777514).

Our research focuses on cell biology, molecular biology, and computational biology, with particular emphasis on tumor drug resistance, cell signaling pathways, the development of fluorescence-tagged cells, _in silico_ simulations, and the automation of cell image detection, tracking, and analysis.

This GitHub organization is home to our approaches to diverse research problems and tools developed my members of Labsinal.

## Notable Repositories

> Some of them are older than this organization profile, so they are stored directly on the main developer profile.

- [esiCancer - Evolutionary _in silico_ Cancer Simulator](https://github.com/bernardohenz/esiCancer) is a cell-autonomous software tool to model evolution of cancer. This program uses real genomic information of tumors to construct a biallelic genome and applies stochastic events, such as mutations, indels and other alterations to this genome. Each event can produce several alterations to the probability of division, death, event rate per division, and/or maximal divisions, which impact the population of cells over time. You can read the paper here: [esiCancer: Evolutionary In Silico Cancer Simulator Free](https://doi.org/10.1158/0008-5472.CAN-17-3924)

- [SCOUTS](https://github.com/jfaccioni/scouts) - Single-Cell Outlier Selector - is a tool that quickly finds OUTLIERS in single-cell datasets, especially those obtained by mass cytometry (CyToF) and single-cell RNA sequencing (scRNA-Seq) platforms. Outliers are often ignored or even removed from data analysis. In cancer, however, single outlier cells can be of major importance, since they have uncommon characteristics that may confer them tolerance or resistance to therapy. You can read the preprint here: [SCOUT: Single-cell outlier analysis in cancer](https://doi.org/10.1101/2020.03.25.007518)

- [CloVarS](https://github.com/jfaccioni/clovars) - Clonal Variability Simulator - is a _in silico_ simulator that generates synthetic single-cell lineage data, as normally obtained from time-lapse microscopy simulation. It can fit simulation parameters from experimental data as well as generate and analyse the simulations. You can read the preprint here: [CloVarS: a simulation of single-cell clonal variability](https://doi.org/10.1101/2024.02.22.581631)

- [DynaFit](https://github.com/jfaccioni/dynafit/) - Dynamic cell fitness analysis through clonogenic variance - is a methodological approach to measure cell fitness dynamics. DynaFit's rationale is based on the growth rate of individual colonies and its variance across colonies with similar size. You can read the paper here: [Cancer Cell Fitness is Dynamic](https://doi.org/10.1158/0008-5472.CAN-20-2488)

---

> üáßüá∑ Vers√£o em portugu√™s

O [Labsinal ‚Äì Laborat√≥rio de Sinaliza√ß√£o e Plasticidade Celular](https://www.ufrgs.br/labsinal/), da [Universidade Federal do Rio Grande do Sul (UFRGS)](https://www.ufrgs.br/site/), √© um grupo de pesquisa multidisciplinar formado por biotecnologistas, biom√©dicos, bi√≥logos e bi√≥logos marinhos, sob a lideran√ßa do [Prof. Dr. Guido Lenz](http://lattes.cnpq.br/4178667286777514).

Nossa pesquisa √© focada em biologia celular, biologia molecular e biologia computacional, com √™nfase especial em resist√™ncia tumoral a f√°rmacos, vias de sinaliza√ß√£o celular, desenvolvimento de c√©lulas marcadas por fluoresc√™ncia, simula√ß√µes _in silico_ e automa√ß√£o na detec√ß√£o, rastreamento e an√°lise de imagens celulares.

## Reposit√≥rios Not√°veis

> Alguns s√£o mais antigos que essa organiza√ß√£o do GitHub, estes est√£o guardados diretamente no perfil do principal desenvolvedor do projeto.

- [esiCancer](https://github.com/bernardohenz/esiCancer) √© uma ferramenta aut√¥noma para modelar a evolu√ß√£o do c√¢ncer. Este programa utiliza informa√ß√µes gen√¥micas reais de tumores para construir um genoma bial√©lico e aplica eventos estoc√°sticos, como muta√ß√µes, indels e outras altera√ß√µes nesse genoma. Cada evento pode causar diversas modifica√ß√µes na probabilidade de divis√£o, morte, taxa de eventos por divis√£o e/ou n√∫mero m√°ximo de divis√µes, o que impacta a popula√ß√£o de c√©lulas ao longo do tempo. Voc√™ pode ler o artigo aqui: [esiCancer: Evolutionary In Silico Cancer Simulator Free](https://doi.org/10.1158/0008-5472.CAN-17-3924)

- [SCOUTS](https://github.com/jfaccioni/scouts) ‚Äì _Single-Cell Outlier Selector_ ‚Äì √© uma ferramenta que identifica rapidamente _outliers_ em conjuntos de dados de c√©lula √∫nica, especialmente aqueles obtidos por citometria de massa (CyToF) e sequenciamento de RNA de c√©lula √∫nica (scRNA-Seq). Valores extremos s√£o frequentemente ignorados ou at√© mesmo removidos da an√°lise de dados. No c√¢ncer, no entanto, c√©lulas √∫nicas com caracter√≠sticas fora do comum podem ter grande import√¢ncia, pois essas caracter√≠sticas podem conferir toler√¢ncia ou resist√™ncia √† terapia. Voc√™ pode ler o preprint aqui: [SCOUT: Single-cell outlier analysis in cancer](https://doi.org/10.1101/2020.03.25.007518)

- [CloVarS](https://github.com/jfaccioni/clovars) ‚Äì _Clonal Variability Simulator_ ‚Äì √© um simulador _in silico_ que gera dados sint√©ticos de linhagens de c√©lulas √∫nicas, como os obtidos por experimentos de microscopia _time-lapse_. Ele pode ajustar os par√¢metros da simula√ß√£o a partir de dados experimentais, bem como gerar e analisar as simula√ß√µes. Voc√™ pode ler o preprint aqui: [CloVarS: a simulation of single-cell clonal variability](https://doi.org/10.1101/2024.02.22.581631)

- [DynaFit](https://github.com/jfaccioni/dynafit/) ‚Äì An√°lise de din√¢mica de _fitness_ celular por meio da vari√¢ncia clonog√™nica ‚Äì √© uma abordagem metodol√≥gica para medir a din√¢mica da aptid√£o celular. A l√≥gica do DynaFit se baseia na taxa de crescimento de col√¥nias individuais e na sua variabilidade entre col√¥nias de tamanho semelhante. Voc√™ pode ler o artigo aqui: [Cancer Cell Fitness is Dynamic](https://doi.org/10.1158/0008-5472.CAN-20-2488)
